crates/core/src/workflows/get_link_result.rs

```diff
@@ -36,6 +41,16 @@ pub async fn get_link_result_workflow<'a>(
         GetLinksNetworkResult::Links(links) => {
             let get_links_result = links
                 .into_iter()
+                .map(|(link_add_address, tag)| {
+                    // make DHT calls to get the entries for the links
+                    get_link_data_from_link_addresses(
+                        context,
+                        &link_add_address,
+                        &tag,
+                        link_args.options.headers,
+                    )
+                    .unwrap()
```

Comment:
This can't be an unwrap!  I'm pretty sure that's where the panic I was was comming from.